packages/tools/locales/es_ES.po

```diff
@@ -3942,326 +3846,315 @@ msgstr "Marca una tarea como hecha."
+msgstr "Importa un archivo de libreta de Evernote (archivo .enex)."
 
-#~ msgid "Imports an Evernote notebook file (.enex file)."
-#~ msgstr "Importar una libreta de Evernote (archivo .enex)."
+msgid "File \"%s\" will be imported into existing notebook \"%s\". Continue?"
+msgstr ""
+"El archivo «%s» será importado a la libreta «%s» existente. ¿Continuar?"
 
-#~ msgid ""
-#~ "File \"%s\" will be imported into existing notebook \"%s\". Continue?"
-#~ msgstr ""
-#~ "El archivo «%s» será importado dentro de la libreta existente «%s». "
-#~ "¿Continuar?"
+msgid ""
+"New notebook \"%s\" will be created and file \"%s\" will be imported into "
+"it. Continue?"
+msgstr ""
+"Nueva libreta «%s» será creada y el archivo «%s» será importado a ella. "
+"¿Continuar?"
 
-#~ msgid ""
-#~ "New notebook \"%s\" will be created and file \"%s\" will be imported into "
-#~ "it. Continue?"
-#~ msgstr ""
-#~ "Nueva libreta «%s» será creada y el archivo «%s» será importado dentro de "
-#~ "ella. ¿Continuar?"
+msgid "Import Evernote notes"
+msgstr "Importar notas de Evernote"
 
-#~ msgid "Import Evernote notes"
-#~ msgstr "Importar notas de Evernote"
+msgid "Give focus to next pane"
+msgstr "Enfocar el siguiente panel"
 
-#~ msgid "Give focus to next pane"
-#~ msgstr "Enfocar el siguiente panel"
+msgid "Give focus to previous pane"
+msgstr "Enfocar el panel anterior"
 
-#~ msgid "Give focus to previous pane"
-#~ msgstr "Enfocar el panel anterior"
+msgid "Exit command line mode"
+msgstr "Salir del modo línea de comandos"
 
-#~ msgid "Exit command line mode"
-#~ msgstr "Salir del modo línea de comandos"
+msgid "Edit the selected note"
+msgstr "Editar la nota seleccionada"
 
-#~ msgid "Cancel the current command."
-#~ msgstr "Cancelar el comando actual."
+msgid "Cancel the current command."
+msgstr "Cancelar el comando actual."
 
-#~ msgid "Set a to-do as completed / not completed"
-#~ msgstr "Marca una tarea como completada/no completada"
+msgid "Set a to-do as completed / not completed"
+msgstr "Marcar una tarea como completada/no completada"
 
-#~ msgid "[t]oggle [c]onsole between maximized/minimized/hidden/visible."
-#~ msgstr ""
-#~ "in[t]ercambia la [c]onsola entre maximizada/minimizada/oculta/visible."
+msgid "[t]oggle [c]onsole between maximized/minimized/hidden/visible."
+msgstr "in[t]ercambia la [c]onsola entre maximizada/minimizada/oculta/visible."
 
-#~ msgid "[t]oggle note [m]etadata."
-#~ msgstr "in[t]ercambia los [m]etadatos de una nota."
+msgid "[t]oggle note [m]etadata."
+msgstr "in[t]ercambia los [m]etadatos de una nota."
 
-#~ msgid "[M]ake a new [n]ote"
-#~ msgstr "[C]rear una [n]ota nueva"
+msgid "[M]ake a new [n]ote"
+msgstr "[C]rear una [n]ota nueva"
 
-#~ msgid "[M]ake a new [t]odo"
-#~ msgstr "[C]rear una [t]area nueva"
+msgid "[M]ake a new [t]odo"
+msgstr "[C]rear una [t]area nueva"
 
-#~ msgid "[M]ake a new note[b]ook"
-#~ msgstr "[C]rear una li[b]reta nueva"
+msgid "[M]ake a new note[b]ook"
+msgstr "[C]rear una li[b]reta nueva"
```

Comment:
I don't think this will work. I think the characters in brackets cannot change and have to be the same.

@laurent22 or does the cli app parse the string and use the characters in brackets as commands?

Reply:
No, those are hard-coded shortcuts. But anyway, it looks like these strings aren't even in the app anymore so they shouldn't be translated.

Reply:
@Elaborendum can you please explain how you re-added the commented out translations? They should not even show up in Poedit.

Reply:
I didn't know how to download individual files, so I copied the raw content from the es_ES.po file and pasted it on a text file, but instead of saving it as .po I saved it as .pot. When I opened it I realized my mistake, but clicked on "Create new translation" thinking it would solve the problem. 

Just now, I tried it saving it as .po from the start and those never showed up.

Reply:
An chance you can revert the changes done to all lines after 3944? Also, you will have to fix the conflict, although I can help with the conflict problem if needed.

Reply:
Sure, I'll do what I can. The conflict's there too so maybe that helps.

Reply:
Thanks, that did it.